src/dpos/components/ValidatorUpdateForm.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <b-modal ref="validatorUpdateform" id="validator-update-form">
+    <template slot="modal-title">
+      {{ $t("components.validator_update_form.title") }}
+    </template>
+    <form>
+      <p class="mb-1">{{ $t("components.validator_update_form.name") }}</p>
+      <b-form-input
+        class="mb-3"
+        id="input-name"
+        v-model="form.name"
+        type="text"
+        required
+      ></b-form-input>
+      <p class="mb-1">
+        {{ $t("components.validator_update_form.description") }}
+      </p>
+      <b-form-input
+        class="mb-3"
+        id="input-description"
+        v-model="form.description"
+        type="text"
+        required
+      ></b-form-input>
+      <p class="mb-1">{{ $t("components.validator_update_form.website") }}</p>
+      <b-form-input
+        class="mb-3"
+        id="input-website"
+        v-model="form.website"
+        type="text"
+        required
+      ></b-form-input>
+      <p class="mb-1">{{ $t("components.validator_update_form.fee") }}</p>
+      <b-form-input
+        id="input-fee"
+        v-model="form.fee"
+        type="number"
+        required
+        :disabled="!ableToChangeFee"
```

Comment:
Can you also add a tooltip to this field that will be displayed when the field is disabled (show a small info icon next to the field that will act as the trigger/target for the tooltip). The tooltip should say `Update already in progress, please try changing the fee after the next election.`